src/plugins/condition/ConditionManager.js

```diff
@@ -410,11 +399,13 @@ export default class ConditionManager extends EventEmitter {
     return this.openmct.time.getBounds().end >= currentTimestamp;
   }
 
-  telemetryReceived(endpoint, datum) {
+  telemetryReceived(endpoint, data) {
     if (!this.isTelemetryUsed(endpoint)) {
       return;
     }
 
+    const datum = data[0];
```

Comment:
What happens if `data[0]` is undefined?

Reply:
We didn't have any handling before, so same as before (probably break). I'm not sure if that's what we'd want to happen or not. I can move the assignment before the conditional above and return if it's undefined?

Reply:
ok, actually i see how this is used now and it seems like we can reasonably assume that `data` will be an array of telemetry with size of at least 1, so we can leave this as-is